src/Features/CSharp/Portable/Completion/CompletionProviders/SymbolCompletionProvider.cs

```diff
@@ -65,6 +128,24 @@ protected override async Task<bool> IsSemanticTriggerCharacterAsync(Document doc
             return token.Kind() != SyntaxKind.NumericLiteralToken;
         }
 
+        private async Task<bool?> IsTriggerInArgumentListAsync(Document document, int characterPosition, CancellationToken cancellationToken)
+        {
+            var text = await document.GetTextAsync(cancellationToken).ConfigureAwait(false);
+            if (!CompletionUtilities.IsArgumentListCharacter(text[characterPosition]))
+            {
+                return null;
+            }
+
+            var tree = await document.GetSyntaxRootAsync(cancellationToken).ConfigureAwait(false);
+            var token = tree.FindToken(characterPosition);
```

Comment:
make sure the positoin is actually the start-pos of this token.  otherwise we can be in a case where we're *inside* the trivia on one of these tokesn, and this will end up matching, even though the user is in a comment (also a good test) :)

Reply:
If anything, it will be the end position of the token, right?

Reply:
not necessarily.  if you type in a comment that precedes a token then you'll find the token that comes after you.  i.e.:

```
// i'm here$$ boi
public
```

FindToken there will find `public`.

Reply:
Added a check for this.